langchain/src/embeddings/voyage.ts

```diff
@@ -0,0 +1,157 @@
+import { chunkArray } from "../util/chunk.js";
+import { getEnvironmentVariable } from "../util/env.js";
+import { Embeddings, EmbeddingsParams } from "./base.js";
+
+enum VoyageModelNames {
```

Comment:
Ah we can't use `enum` unfortunately as it'll break tree shaking :(

Can you use a type here instead?

Reply:
@jacoblee93 i removed even the type, cause i see on the Voyage docs, that they have more embeddings model rolling out, so i'll just leave the possibility tu change the model name :)